tests/functional/adapter/dynamic_table_tests/test_dynamic_tables_changes.py

```diff
@@ -131,17 +133,16 @@ class TestSnowflakeDynamicTableChangesApply(SnowflakeDynamicTableChanges):
     def project_config_update(self):
         return {"models": {"on_configuration_change": OnConfigurationChangeOption.Apply.value}}
 
-    def test_change_is_applied_via_alter(self, project, adapter, my_dynamic_table):
-        """
-        See above about the two commented assertions. In the meantime, these have been validated manually.
-        """
-        # self.check_start_state(adapter, my_dynamic_table)
+    def test_change_is_applied_via_alter(self, project, my_dynamic_table):
 
+        # update the settings
         self.change_config_via_alter(project, my_dynamic_table)
         _, logs = run_dbt_and_capture(["--debug", "run", "--models", my_dynamic_table.name])
 
-        # self.check_state_alter_change_is_applied(adapter, my_dynamic_table)
+        # verify the updated settings are correct in Snowflake
+        self.check_state_alter_change_is_applied(project, my_dynamic_table)
```

Comment:
We're now checking in the database to verify the settings were applied. Even though that is a stronger check than checking the logs, we still want to check the logs so that we know how the change was made. If a change that was supposed to be applied via an alter is actually applied via a drop/create, that's not passing.